example/ionExample.js

```diff
@@ -99,14 +102,28 @@ function init() {
 
 	scene = new Scene();
 
+	// Add scene light for MeshStandardMaterial to react properly if its metalness is set to 0
+	const light = new AmbientLight( 0xffffff, 1 );
+	scene.add( light );
+
```

Comment:
I'll remove it - but now that we have an env map this ambient like will no longer be needed